app/templates/frameworks/declaration_overview.html

```diff
@@ -72,56 +72,28 @@ <h1 class="govuk-heading-l">Your declaration overview</h1>
   </div>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
-      {% import "toolkit/summary-table.html" as summary %}
+
       {% for section_slug, (section, section_errors) in sections_errors.items() %}
-        {{ summary.heading(section.name, id=section.slug) }}
-        {% if section.editable %}
-          {% if framework.status == 'open' %}
-            {{ summary.top_link("Edit", url_for(".framework_supplier_declaration_edit", framework_slug=framework.slug, section_id=section.id), hidden_text=section.name) }}
+        <h2 class="govuk-heading-m dm-summary-list-heading" id="{{ section.slug }}">
+          {{ section.name }}
+          {% if section.editable %}
+            {% if framework.status == 'open' %}
+              <span class="dm-section-action-link">
+                <a class="govuk-link" href="{{ url_for('.framework_supplier_declaration_edit', framework_slug=framework.slug, section_id=section.id)}}">
+                  Edit
+                  <span class="govuk-visually-hidden">{{ section.name }}</span>
+                </a>
+              </span>
+            {% endif %}
```

Comment:
Interesting, what's the advantage of putting the edit link in the heading like this?

Reply:
Oh gosh, none, other than to allow quick prototyping while developing and then miss when pushing. Sorry about this, will tweak now.